bin/lib/utils/translator-settings.js

```diff
@@ -22,6 +23,7 @@ function getAvailableLanguages(languages) {
  		});
 
  		if(duplicate === -1) {
+			languages[i].isDefault = (languages[i].name === DEFAULT_LANG);
```

Comment:
I would base the default language on the language code to LowerCase, instead of name (spelling could vary)